assets/files/etc/sysconfig/network-scripts/ifcfg-baremetal

```diff
@@ -0,0 +1,5 @@
+DEVICE=baremetal
```

Comment:
In KNI context everything is bare metal, so this bridge name means very little. How about `brext`, making clear that it is a bridge that is connected to the external world?

Reply:
i m using baremetal for the bridge name,  since it s how it s named on the hypervisor side, when using libvirt setup.

Reply:
Names should be set according to how customers see them, not according to what we used in our virtual test environment. What would a customer understand when she sees `baremetal` on her machine?

I personally think that the we should use `ext` in libvirt, and `brext` for the on-host bridge name.
@celebdor @derekhiggins  do you have an opinion about that?

Reply:
`ext` and `brext` feel like the right choice.

Reply:
switched to brext